buzzmobile/tools/state_viz/state_viz.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import cv2
+import os
+import rospy
+import sys
+
+from buzzmobile.msg import CarPose
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+from time import sleep
+
+
+g = {} # globals
+g['start'] = g['auto'] = g['manual'] = g['neutral'] = None
```

Comment:
These are initial variables being set to `None` so I don't think there's an issue because, IIRC,  anything that `is None` is pointing to the same `None` in memory anyways.